addons/l10n_it_edi/data/ir_cron.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record id="ir_cron_receive_fattura_pa_invoice" model="ir.cron">
-        <field name="name">FatturaPA: Receive invoices from the exchange system</field>
+        <field name="name">IT EDI: Receive invoices from the SdI</field>
```

Comment:
FatturaPA is the name of the XML file format protocol. Like, the name of the XSD.
SdI is the name of the Tax Agency's system, and will generally be used anywhere for consistency.